data/participants.json

```diff
@@ -448,5 +448,15 @@
             "twitter"  : "",
             "referral" : ""
 	    }
```

Comment:
Check the syntax of json. To check the validity use [this](https://jsonlint.com/)